renderer.php

```diff
@@ -137,7 +137,13 @@ public function formulation_and_controls(question_attempt $qa,
         $activitytype = 'standalone';
         $templateid = $standalonestrip;
 
-        $PAGE->requires->js_call_amd('filter_siyavula/initmathjax', 'init');
+        // Current version is Moodle 4.0 or higher use the event types. Otherwise use the older versions.
+        if ($CFG->version >= 2022041912) {
```

Comment:
Thank you for your comment! I want to make sure I understand your point correctly. Are you suggesting that the version check should be based on the Moodle version instead of the plugin version? If so, I believe that the current check is already using the Moodle version ($CFG->version).

Please clarify if you are referring to something specific, or is there a particular aspect of the version check you think I should address? Your insights would be beneficial!

Reply:
@prasanna-lmsace Sorry, I was confused between the `$CFG->version` and `$plugin->version` 😅 I'll close this one.